src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustReviewDto.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dto.trust;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TrustReviewDto {
+    public static final String IN_REVIEW = "in_review";
```

Comment:
Is the fact of having one of these on the trust not a sign that the Trust is in review? So is this flag actually needed?

Reply:
But I guess that's assuming the first Trust in the appData.update.review_trusts is always the one being reviewed (if there is one).

Reply:
The mapping hasn't been done yet on the frontend, but this would populate review_status when mapping into review_trusts.